packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -83,17 +81,18 @@ export class MenuItem extends FASTElement {
     }
 
     /**
+     * reference to the anchored region
+     *
      * @internal
      */
     @observable
-    public submenu: Element | undefined;
+    public submenuRegion: AnchoredRegion;
```

Comment:
@scomea is this a breaking change?

Reply:
It's marked internal so I'd say that means folks shouldn't be taking dependencies on it, but it is an unnecessary change so I'll revert it to minimize churn